pkg/image/api/types.go

```diff
@@ -81,6 +91,16 @@ type TagReference struct {
 	From *kapi.ObjectReference
 	// Reference states if the tag will be imported. Default value is false, which means the tag will be imported.
 	Reference bool
+	// Generation is the image stream generation that updated this tag - setting it to 0 is an indication that the generation must be updated.
```

Comment:
Self-defining.  What is an image stream generation?  Is it more than just a hack to avoid introspecting swagger?